### PR TITLE
Add Centrifuge - chachacha

### PR DESCRIPTION
**Pull request by mikiquantum**
@BenWhiteJam As discussed, please have a look and check if the wording is clear.

### CODE REVIEW

**Comment by nuke-web3** [comment]
excited to see this as an option - thanks for offering this to the community!

**Comment by mikiquantum** [comment]
@NukeManDan If I resolve the failing check running `node scripts/update-substrate.js` it brings up other changes. How should I proceed?

File: docs/build-parachains-rococo.md [hunk 1]
```diff
@@ -202,3 +202,67 @@ _Update: 15. Jan. 2021_
...
+  further performance and stability testing
+- Parachains to find issues beforehand, by integrating earlier.
+
+## Characteristics
+
+- Chachacha will be at par with the latest Rococo polkadot/cumulus/substrate version
+- The network will be refreshed and restarted frequently
+- Inclusion process is analogous to Rococo's
+- Notifications and Support will be given in the Rococo and Rococo Validator Lounge Element channels:
+    - Tag your question using the following user handle: @mikiquantum:matrix.org
```
Comment (nuke-web3) [inline docs/build-parachains-rococo.md]: Please don't bootstrap on our rococo channel. I would much rather have you support and maintain a dedicated one.

**Review by nuke-web3 — CHANGES_REQUESTED** [review]

File: docs/build-parachains-rococo.md (hunk 1 above)
Comment (nuke-web3) [reply docs/build-parachains-rococo.md]: it's fine to have people be in both, but you have a separate network here, just happens to be based on the same codebase :)

**Comment by mikiquantum** [comment]
@NukeManDan Updated and added dedicated channel.

**Comment by nuke-web3** [comment]
@mickstar LGTM!

**Comment by Swader** [comment]
Needs prettify, then it's fine to merge. `npx prettify --save` should do IIRC.

**Comment by mikiquantum** [comment]
@Swader I didn't find `prettify` in the npm dependencies. I ran this instead:
```
npx prettier --write docs/build-parachains-rococo.md
```
if that is not enough, would be good if you can point me to the correct way to install the prettify package.

**Comment by Swader** [comment]
That's it, thanks!

**Pull request merged**